pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -876,6 +941,7 @@ func (r *ReconcileHyperConverged) getAllResources(cr *hcov1alpha1.HyperConverged
 		newKubeVirtCommonTemplateBundleForCR(cr, OpenshiftNamespace),
 		newKubeVirtNodeLabellerBundleForCR(cr, request.Namespace),
 		newKubeVirtTemplateValidatorForCR(cr, request.Namespace),
+		newIMSConfigForCR(cr, request.Namespace),
```

Comment:
This is recently resolved to `kubevirt-hyperconverged`, which is ok. But we need to be careful if the deployment context changes.

Reply:
Can the v2v operator run in any namespace?  Does the namespace need to be known ahead of time by another party?

Reply:
Keep in mind that the downstream namespace is `openshift-cnv`.

Reply:
Thanks Ryan, Let me get back to you on whether they are aware of this.

Reply:
> Keep in mind that the downstream namespace is `openshift-cnv`.

Will this be the same for RHHI.next?

Reply:
yes,  `openshift-cnv` is the cnv 2.1 namespace.  RHHI is using CNV 2.1

Reply:
It sounds like a patch will  be pushed to IMS to handle the different namespaces for upstream/downstream. Thanks for raising this @rthallisey